src/features/flows/controllingConfig.ts

```diff
@@ -275,6 +275,74 @@ export const controllingConfig: ControllingConfig[] = [
             },
         ],
     },
+    {
+        name: 'Matter Contact Sensor',
+        controllingGuide: [
+            {
+                name: 'Apple Home',
+                guide: [
+                    'Press the button marked with green rectangle on the DK',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns on</b> while pressing the button',
+                    'Release the button',
+                    'Observe that <b>LED</b> marked with green rectangle on the DK <b>turns off</b>',
+                    'Open the <b>Apple Home</b> app',
+                    'Navigate to the room that contains the device that was previously paired',
+                    'Tap the <b>Activity History</b> tile to open a list of events',
+                    'Press the button marked with green rectangle on the DK once again',
+                    'Wait for the update in the <b>Apple Home</b> app and observe that the event is added to the <b>Activity History</b> list as <bClosed/b>',
+                    'Release the button',
+                    'Wait for the update in the <b>Apple Home</b> app and observe that the event is removed from the <b>Activity History</b> list as <bOpened/b>',
+                ],
+                video: '../resources/ecosystems/Apple/usage/apple_contact_usage.mp4',
+                videoDeviceName: 'Matter Accessory',
```

Comment:
What is this `videoDeviceName` for? I can see we have a bit mess with naming them across all samples and ecosystems.

Reply:
Actually, you have added this field here, so you can explain. :smile: